exposee-app/Routes/users.js

```diff
@@ -70,11 +70,12 @@ router.post('/user/login', async (req, res) => {
     if (!isValidPassword) {
       return res.status(401).json({ error: 'Invalid username or password' });
     }
-    const access_Token = sign({username : user.username, id:user.id}, "iamtheSwat1*");
-    const responsePayload = {
-      user: user,
-      access_token: access_Token,
-    };
+    const access_Token = sign({ username: user.username, id: user.id }, "iamtheSwat1*");
+  const responsePayload = {
+    user: user,
+    access_token: access_Token,
+    user_id: user.id, 
```

Comment:
I'm not sure I understand what adding user_id is for, could we not just call user.id when it's needed?